forwards_paper.tex

```diff
@@ -195,15 +194,21 @@ \section*{Results}
 %%%%%%
 \subsection*{Simulation benchmarks}
 
-To measure the performance gains from recording the pedigree we ran simulations with and
-without recording, for both implementations (\simupop{} \citep{simupop}
-and \fwdpp{} \citep{fwdpp}).
+To measure the performance gains from recording the pedigree we ran simulations both with and without recording.
 All simulations implemented a discrete-time Wright-Fisher population of diploid
 population size $N$, simulated for $10N$ generations.
-The simulations without pedigree recording introduced neutral mutations at a scaled mutation
-rate equal to the scaled recombination rate (\textit{e.g.}, $\theta = 4N\mu = \rho$, where $\rho = 4Nr$, $\mu$ is the
-mutation rate to neutral mutations (per gamete, per generation) and $r$ is the recombination rate per diploid per
-generation.
+The simulations without pedigree recording introduced neutral mutations at a rate 
+equal to the recombination rate,
+so $\mu = r$, where $\mu$ and $r$ are the expected number of mutations and recombinations,
+respectively, per gamete and per generation.
+% Both mutations and recombinations occur as Poisson processes with respective means $\mu$ and $r$,
+% and we used the infinitely-many sites mutation model \citep{Kimura1969-uc}.
+We ran simulations with different values of $N$
+and of the \emph{scaled} parameter $\rho = 4Nr$,
+categorized by the value of $\rho$, since this
+is what most strongly determines run time
+(along with the scaled mutation rate, $\theta = 4N\mu$, which is also equal to $\rho$.)
```

Comment:
I think I've left in all the important information.  We don't need to list parameter combinations in the text, as they will appear in the figures/tables, I figured.